### ISSUE TITLE
lower case, false positives

### ISSUE THREAD

**Issue by larshp** [larshp/abaplint#341]
```abap
    CALL FUNCTION 'ZMOOBOO'
      EXPORTING
        iv_fild     = lv_value
      EXCEPTIONS
        invalid_boo = 1
        OTHERS      = 2.


CLASS ZCL_ABAPGIT_ZLIB_STREAM IMPLEMENTATION.
```
